### ISSUE TITLE
[KED-2611] CLI --pipeline arg throws KedroContext attribute error

### ISSUE THREAD

**Issue by edwardcjohnson** [quantumblacklabs/kedro-viz#433]
## Description
The --pipeline argument throws an error. It seems the latest version of kedro has no attribute '_get_pipeline' under KedroContext.
Reference:
https://github.com/quantumblacklabs/kedro/blob/c5ccb630c4da17c22f699d35777e36877c088379/kedro/framework/cli/utils.py

## Context
I am trying to view a specific pipeline

## Steps to Reproduce
1. Define a kedro pipeline and give it a name
2. verify that the specific pipeline runs successfully in kedro using: `kedro run --pipeline <pipeline_name>`
3. Run kedro viz to launch UI: `kedro viz --pipeline <pipeline_name>`

## Expected Result
A kedro viz server should be launched to view the specific pipeline

## Actual Result
Error message:
```
kedro.framework.cli.utils.KedroCliError: 'KedroContext' object has no attribute '_get_pipeline'
Run with --verbose to see the full exception
Error: 'KedroContext' object has no attribute '_get_pipeline'
```

## Your Environment

* Web browser: Chrome
* Operating system and version: Ubuntu 16.04.7
* Kedro version: 0.17.3
* kedro-viz version: 3.11.0
* Python version: 3.8.8

## Checklist
- [ ] Include labels so that we can categorise your issue

**Comment by limdauto** [comment]
Hi @edwardcjohnson thank you for reporting the issue. This has been logged and will be fixed in the next release.

**Comment by limdauto** [comment]
This should be fixed now in kedro-viz 3.12.0!
